src/connector/src/source/cdc/external/mysql.rs

```diff
@@ -107,13 +107,15 @@ impl MySqlExternalTable {
         let mut pk_names = vec![];
         for col in columns {
             let data_type = type_to_rw_type(&col.col_type)?;
+            // column name in mysql is case-insensitive, convert to lowercase
+            let col_name = col.name.to_lowercase();
```

Comment:
It really depends on the underlying file system
https://dev.mysql.com/doc/refman/8.4/en/identifier-case-sensitivity.html
> Consequently, the case sensitivity of the underlying operating system plays a part in the case sensitivity of database, table, and trigger names. This means such names are not case-sensitive in Windows, but are case-sensitive in most varieties of Unix. One notable exception is macOS, which is Unix-based but uses a default file system type (HFS+) that is not case-sensitive. However, macOS also supports UFS volumes, which are case-sensitive just as on any Unix.

It does not mention macOS APFS above, which can actually be formatted as either sensitive or insensitive 😂 

What is the motivation of this change? According to the doc above usage on Linux is supposed to be sensitive. Although practically I guess no people would want to differentiate 2 columns by cases only.

Reply:
We have checked the page before. column name is case insensitive across all platform.

> Partition, subpartition, column, index, stored routine, event, and resource group names are not case-sensitive on any platform, nor are column aliases.